src/structures/GuildChannel.js

```diff
@@ -213,11 +213,19 @@ class GuildChannel extends Channel {
     return this;
   }
 
+  /**
+   * Extra information about the overwrite
+   * @typedef {Object} ExtraOverwriteInfo
```

Comment:
```suggestion
   * @typedef {Object} GuildChannelOverwriteOptions
```

Reply:
I'm not sure about this name, it's used in both `createOverwrite` and `updateOverwrite`.

cc: @vladfrangu @iCrawl

Reply:
👍 for `GuildChannelOverwriteOptions`